staging/src/k8s.io/component-base/config/v1alpha1/defaults.go

```diff
@@ -19,7 +19,6 @@ package v1alpha1
 import (
 	"time"
 
-	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
Will TODO(#80289) be addressed in this PR like it was being addressed in #105448?

Reply:
No, this PR is just about LoggingConfiguration. Once it has gone through API review and got merged, similar changes can and probably should be done for the other structs that are currently in component-base/config.